yarn-project/circuits.js/src/scripts/constants.in.ts

```diff
@@ -229,7 +229,7 @@ function generateCppConstants({ constants }: ParsedContent, targetPath: string)
 #pragma once
 
 ${processConstantsCpp(constants)}
-\n`;
```

Comment:
This was a minor bug -- there were 2 new lines inserted at the end of the file aztec-constants.hpp file.